_data/team/roles/code_of_conduct.yml

```
@@ -1,11 +1,12 @@
 role: Code of conduct committee
 description: |
-  The [code of conduct committee](https://www.mdanalysis.org/pages/conduct/) members
-  are elected by the [project leadership](https://www.mdanalysis.org/about/#governance) when a committee members' term limit (2 years) has expired.
+  The [code of conduct](https://www.mdanalysis.org/conduct/) committee members
+  are elected by the [project leadership](https://www.mdanalysis.org/pages/governance/) when a committee members' term limit (2 years) has expired.
 tasks: 
-  - Receive and investigate all code of conduct complaints and violations
-  - Propose actions and sanctions, as necessary, to the [project leadership](https://www.mdanalysis.org/about/#governance) (project leadership is tasked with enforcement)
-  - Periodically review, revise and update current code of conduct procedures
+  - Ensure that the application of the code of conduct remains up-to-date, reflecting any changes from the NumFOCUS and MDAnalysis communities
+  - Coordinate with and defer to the [NumFOCUS Code of Conduct Working Group](https://www.mdanalysis.org/conduct/#who-will-receive-your-report) for complaints/violation reports and initial recommendations
+  - Propose actions and sanctions, as necessary, to the [project leadership](https://www.mdanalysis.org/pages/governance/), original reporter, and any other affeccted parties (project leadership is tasked with enforcement)
```

`{{ site.baseurl }}`
```suggestion
  - Propose actions and sanctions, as necessary, to the [project leadership]({{ site.baseurl }}/pages/governance/), original reporter, and any other affeccted parties (project leadership is tasked with enforcement)
```